tests/adapters_t/sites_t/test_htcondorsiteadapter.py

```diff
@@ -143,28 +143,28 @@ def test_resource_status_run(self):
         self.assertEqual(response.resource_status, ResourceStatus.Running)
 
     @mock_executor_run_command(stdout=CONDOR_Q_OUTPUT_REMOVING)
-    def test_resource_status_idle(self):
+    def test_resource_status_removing(self):
         response = run_async(
             self.adapter.resource_status, AttributeDict(remote_resource_uuid="1351043")
         )
         self.assertEqual(response.resource_status, ResourceStatus.Running)
 
     @mock_executor_run_command(stdout=CONDOR_Q_OUTPUT_COMPLETED)
-    def test_resource_status_idle(self):
+    def test_resource_status_completed(self):
         response = run_async(
             self.adapter.resource_status, AttributeDict(remote_resource_uuid="1351043")
         )
-        self.assertEqual(response.resource_status, ResourceStatus.Stopped)
+        self.assertEqual(response.resource_status, ResourceStatus.Deleted)
```

Comment:
Please check this carefully. I'm not sure if this is correct.

Reply:
@giffels, this I also cannot tell. Please have a look :) Otherwise I am fine with this PR.

Reply:
Thank you very much for fixing this. The behaviour above is correct. Looks good to go!